src/checks.rs

```diff
@@ -26,8 +27,11 @@ trait Check {
     fn run(&self, line: &LineEntry) -> Option<Warning>;
 }
 
-fn checklist() -> Vec<impl Check> {
-    vec![leading_space::LeadingSpaceChecker::default()]
+fn checklist() -> Vec<Box<dyn Check>> {
+    vec![
+        Box::new(leading_space::LeadingSpaceChecker::default()),
+        Box::new(incorrect_delimiter::IncorrectDelimiterChecker {}),
```

Comment:
Please, implement the trait `Default` for this struct and use the method `default`:
```suggestion
        Box::new(incorrect_delimiter::IncorrectDelimiterChecker::default()),
```